src/FormatCommand.cs

```diff
@@ -75,6 +77,11 @@ internal static RootCommand CreateCommandLineOptions()
                 {
                     IsHidden = true
                 },
+                new Option(new[] {"--binarylog", "-bl" }, Resources.Log_all_project_or_solution_load_information_to_a_binary_log_file)
+                {
+                    Argument = new Argument<string?>(
+                        () => Path.Combine(Environment.CurrentDirectory, "formatDiagnosticLog.binlog") ) { Name = "binary-log-path" }.LegalFilePathsOnly()
```

Comment:
This doesn't set it up to always provide a binarylog path right?

Reply:
my understanding is that this _should_ make it so `--binarylog` without a path just puts it in the current directory. I'll add a test for this

Reply:
added tests and updated this